plugins/rest/help.md

```diff
@@ -12,459 +11,320 @@ This plugin is often used to integrate with ad-hoc 3rd party API's in a workflow
   "secret": "9de5069c5afe602b2ea0a04b66beb2c0",
   "ssl_verify": true
 }
-```
-
-Example input (with Insight Platform):
-
-```
-{
-  "authentication_type": "Rapid7 Insight",
-  "base_url": "https://us.api.insight.rapid7.com",
-  "secret": {
-    "secretKey": "12341234-1234-1234-1234-123412341234"
-  }
-  "default_headers": {
-    "User-Agent": "Rapid7 InsightConnect"
-  },
-  "fail_on_http_errors": false,
-  "ssl_verify": true
-}
-```
+```  
 
 ## Technical Details
 
 ### Actions
 
-#### PUT
-
-This action is used to make a PUT request.
+#### DELETE
+  
+Make a DELETE request  
 
 ##### Input
 
 |Name|Type|Default|Required|Description|Enum|Example|
-|----|----|-------|--------|-----------|----|-------|
+| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
 |body_any|string|None|False|Payload (string) to submit to the server when making the HTTP Request call. This can be any type of input, such as an array or integers etc.. If a data object is to be sent, please use the 'Body Object' input|None|test data|
 |body_object|object|None|False|Payload to submit to the server when making the HTTP Request call|None|{"user": "user@example.com"}|
 |headers|object|None|False|Headers to use for the request. These will override any default headers|None|{"Host": "rapid7.com"}|
 |route|string|None|True|The route to append to the base URL e.g. /org/users|None|/org/users|
-
+  
 Example input:
 
 ```
 {
+  "body_any": "test data",
   "body_object": {
     "user": "user@example.com"
   },
-  "body_any": "a sample data body",
   "headers": {
     "Host": "rapid7.com"
   },
   "route": "/org/users"
 }
-```
+```  
 
 ##### Output
 
-|Name|Type|Required|Description|Example|
-| :--- | :--- | :--- | :--- | :--- |
-|body_object|object|False|Response payload from the server as an object. Note, if the response has invalid object structure (list, string..) plugin will wrap it with object map|{'user': 'user@example.com'}|
-|body_string|string|False|Response payload from the server as a string|test data|
-|headers|object|False|Response headers from the server|{'Host': 'rapid7.com'}|
-|status|integer|False|Status code of the response from the server|200|
-
+|Name|Type|Default|Required|Description|Enum|Example|
```

Comment:
This needs to be updated in the tooling soon. No need to have that fields added.
```suggestion
|Name|Type|Required|Description|Example|
```